.travis.yml

```diff
@@ -33,11 +33,6 @@ stages:
 
 jobs:
   include:
-    - stage: lint
```

Comment:
Why are you deleting this?

Reply:
The new black will fail on this old branch. I still wanted tests to run on this branch, though.